src/gui/GuiWindow.zig

```diff
@@ -356,6 +367,10 @@ pub fn updateHovered(self: *GuiWindow, mousePosition: Vec2f) void {
 }
 
 pub fn updateWindowPosition(self: *GuiWindow) void {
+	if(self.contentSize[0] < iconWidth*4) {
+		std.log.err("Window width is too small: {d}px before scaling", .{self.contentSize[0]});
```

Comment:
I would like to see the window id in there as well, otherwise this is kind of useless.